ocaml/xapi/storage_impl.ml

```diff
@@ -785,9 +803,7 @@ module Wrapper = functor(Impl: Server_impl) -> struct
 
       failure
 
-    let destroy context ~dbg ~dp ~allow_leak =
-      info "DP.destroy dbg:%s dp:%s allow_leak:%b" dbg dp allow_leak;
-      let failures = Host.list !Host.host
+    let destroy context ~dbg ~dp ~allow_leak = info "DP.destroy dbg:%s dp:%s allow_leak:%b" dbg dp allow_leak; let failures = Host.list !Host.host
```

Comment:
minor: I think you didn't actually change anything here, can you put back the lines as there were indented before?